pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/model/nonpepperkit/NonPepperKitCreationService.java

```diff
@@ -24,9 +24,11 @@
 @Slf4j
 public class NonPepperKitCreationService {
     public static final String JUNIPER = "JUNIPER";
+    NonPepperStatusKitService nonPepperStatusKitService;
 
-    //These are the Error Strings that are expected by Juniper
-
+    public NonPepperKitCreationService(NonPepperStatusKitService nonPepperStatusKitService) {
```

Comment:
Do we need invoking services to pass the NonPepperStatusKitService ? 
Can't we just create a instance ?